resources/prosody-plugins/mod_filter_iq_jibri.lua

```diff
@@ -0,0 +1,24 @@
+local st = require "util.stanza";
+
+-- filters jibri iq in case of requested from jwt authenticated session that
+-- has features in the user context, but without feature for recording
+module:hook("pre-iq/full", function(event)
+    local stanza = event.stanza;
+    if stanza.name == "iq" then
+        local jibri = stanza:get_child('jibri', 'http://jitsi.org/protocol/jibri');
+        if jibri then
+            local session = event.origin;
+            local token = session.auth_token;
+
+            if jibri.attr.action == 'start'
+                and jibri.attr.recording_mode == 'file'
+                and session.jitsi_meet_context_user["features"]
+                and session.jitsi_meet_context_user["features"]["recording"] ~= "true" then
+                module:log("info",
+                    "Filtering jibri start recording, stanza:%s", tostring(stanza));
+                session.send(st.error_reply(stanza, "auth", "forbidden"));
```

Comment:
does the existing handling in the UI handle this response alright?  seems like we might want a new error message

Reply:
Well, I didn't care about this case, cause it is not a normal use case. This would happen if someone tries to trigger a recording if its record button is hidden ... Do you think we need to handle it?

Reply:
i think if there's an error for it, we should handle it.  now, if this falls back to the general 'error' handling (meaning it is handled, it's just a generic error message instead of a specific one) then that's probably fine.

Reply:
Yep we handle it and we show: 
<img width="514" alt="screen shot 2018-05-31 at 16 15 09" src="https://user-images.githubusercontent.com/3263098/40808836-14f3926e-64ee-11e8-82e6-4c4268abe799.png">
The only problem is the description, but in case someone is trying to workaround the UI, I suppose this doesn't matter so much.
<img width="444" alt="screen shot 2018-05-31 at 16 15 17" src="https://user-images.githubusercontent.com/3263098/40808862-2d01a512-64ee-11e8-9dd8-2ffb9239ee0e.png">
He will of course try again 😁

Reply:
But I just found another issue in this code ... when there is no token it crashes. I need to update it.

Reply:
Updated, no recording when the filter is enabled and no token provided, the same way we do for filtering rayo messages.

Reply:
just to make sure i understand it right:
1) if the session has a token, then the permissions for recording must be there to let this through
2) if there is no token, then we let the recording request go through no matter what
3) the token is entirely on the prosody side, so there's no way a user could make the token not be there if it should be
right?